nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -280,6 +287,9 @@ class BtActionNode : public BT::ActionNodeBase
       throw std::runtime_error("send_goal failed");
     }
 
+    if (goal_handle_) {
+      goal_handle_.reset();
```

Comment:
This is unnecessary. The goal_handle_ is about to be set to a new value 2 lines down